### PR TITLE
finish task7

### PR DESCRIPTION
**Pull request by jorgemf2604**
This is a PR for Task 7.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,33 +9,33 @@ import { FullArticle } from "./components/FullArticle";
...
+  // const [articles, setArticles] = useState([]);
+  // const [isLoading, setIsLoading] = useState(true);
 
-  const loadingMsg = isLoading ? (
-    <div className="loading-spinner">
-      <h2>Loading articles, please wait</h2>
-      <CircularProgress size={150} />
-    </div>
-  ) : null;
+  // const loadingMsg = isLoading ? (
```
Comment (HannahBurks) [inline src/App.js:15]: Let's remove any commented out code or stray console logs :)

File: src/utils/api.js [hunk 2]
```diff
@@ -21,3 +21,15 @@ export const getCommentsInArticle = (article_id) => {
     return data.comments;
   });
 };
+
+export const modifyVotesInArticle = (article_id, num) => {
+  console.log(article_id);
```
Comment (HannahBurks) [inline src/utils/api.js:26]: Remove console logs such as these :D

**Review by HannahBurks — APPROVED** [review]
Lovely work Jorge, please remove commented out code and console logs and push up again. Once you have done that, you can then merge and move onto next ticket :)

File: src/App.js (hunk 1 above)
Comment (jorgemf2604) [reply src/App.js:15]: Thanks!

**Pull request merged**